man/melt.data.table.Rd

```diff
@@ -87,52 +87,58 @@ An unkeyed \code{data.table} containing the molten data.
 set.seed(45)
 require(data.table)
 DT <- data.table(
-      i_1 = c(1:5, NA),
-      i_2 = c(NA,6,7,8,9,10),
-      f_1 = factor(sample(c(letters[1:3], NA), 6, TRUE)),
-      f_2 = factor(c("z", "a", "x", "c", "x", "x"), ordered=TRUE),
-      c_1 = sample(c(letters[1:3], NA), 6, TRUE),
-      d_1 = as.Date(c(1:3,NA,4:5), origin="2013-09-01"),
-      d_2 = as.Date(6:1, origin="2012-01-01"))
+  i_1 = c(1:5, NA),
+  n_1 = c(NA, 6, 7, 8, 9, 10),
```

Comment:
`i_2` was actually `numeric` so `i` is a bit confusing. further it's useful to have a `numeric` column (see below), so not just converting to `integer`